src/main/java/uk/gov/pay/card/model/CardInformation.java

```diff
@@ -41,8 +35,10 @@ public CardInformation(String brand, String type, String label, Long min, Long m
         this.corporate = corporate;
         this.prepaidStatus = prepaidStatus;
 
-        if (brand != null) {
-            this.brand = brandMapping.getOrDefault(brand, brand.toLowerCase());
+        if ( brand != null ) {
```

Comment:
No spaces after the opening and before the closing parentheses please! (Use `(brand != null)` instead.)

Reply:
argh, too much time with maven's coding standards